lib/runtime.py

```diff
@@ -286,8 +286,11 @@ def GetClient():
 
     master, myself = ssconf.GetMasterAndMyself(ss=ss)
     if master != myself:
-      raise errors.OpPrereqError("This is not the master node, please connect"
-                                 " to node '%s' and rerun the command" %
-                                 master, errors.ECODE_INVAL)
+      # print our own name to aid debugging, esp. re FQDN
+      # common error is: hostname != hostname.domain.tld
+      raise errors.OpPrereqError("This node, '%s', is not the master node,
+                                 " please connect to node '%s' and rerun "
+                                 " the command" %
+                                 % (myself, master), errors.ECODE_INVAL)
```

Comment:
In such cases I personally prefer f-strings
```suggestion
      raise errors.OpPrereqError(f"This node, {myself}, is not the master node, "
                                 f"please connect to node {master} and rerun "
                                 "the command",
                                 errors.ECODE_INVAL)
```
Perhaps the old %-style should be replaced by f-strings, which are supported in version 3.6?

Reply:
Thanks for your suggestion. f-strings look pretty readable. However for the moment we stay with the old `%` style to align with existing code. Maybe in future we can adopt to follow the [language authorities standards](https://peps.python.org/pep-0498/).